images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -630,3 +630,56 @@ function Get-GitHubPackageDownloadUrl {
     return $downloadUrl
 }
 
+function Use-ChecksumСomparison {
+    param (
+        [Parameter(Mandatory=$true)]
+        [string]$localFileHash,
+        [Parameter(Mandatory=$true)]
+        [string]$distributorFileHash
+    )
+
+    Write-Verbose "Performing checksum verification"
+
+    if ($localFileHash -ne $distributorFileHash) {
+        Write-Host "hash must be equal to: ${distributorFileHash}"
+        Write-Host "actual hash is: ${localFileHash}"
+        throw 'Checksum verification failed, please rerun install'
+    }else{
+        Write-Host "Checksum verification passed"
+    }
+}
+
+function Get-HashFromGitHubReleaseBody {
+    param (
+        [string]$RepoOwner,
+        [string]$RepoName,
+        [Parameter(Mandatory=$true)]
+        [string]$FileName,
+        [string]$Url,
+        [string]$Version = "latest",
+        [boolean]$IsPrerelease = $false,
+        [int]$SearchInCount = 100,
+        [string]$Delimiter = ' | ',
```

Comment:
Is it possible to remove spaces in `$Delimiter`, and `Trim()` in `$result` instead?